app/javascript/components/rooms/RoomsList.jsx

```diff
@@ -32,11 +36,18 @@ export default function RoomsList() {
         </Col>
       </Row>
       <Row md={4} className="g-4">
-        {rooms.map((room) => (
-          <Col key={room.friendly_id}>
-            <RoomCard id={room.friendly_id} name={room.name}> </RoomCard>
-          </Col>
-        ))}
+        {
+          rooms.filter((room) => {
```

Comment:
Hmm, are we going fully client side on this?
At least we can configure the `getRooms` query `interval` maybe to quasi-synchronize the rooms list in the UI with the actual data or consider maybe invalidating the query with each search (which may have some performance implication so at least debounce it).
We may keep it like this for this milestone, but we need to come back to it later.